insights/mappers/systemctl_show.py

```diff
@@ -23,10 +74,19 @@ def parse_content(self, content):
         We will also suppressed empty properties in return data.
         """
         data = {}
+        data = split_kv_pairs(content, use_partition=False, ordered=True)
+
+        """Remove empty key"""
+        self.data = {k: v for k, v in data.iteritems() if not v == ''}
```

Comment:
If the sequence of the data is important, indicated by `ordered=True` above, you may want to consider creating a list of tuples and then using that to initialize an `OrderedDict`.